moveit_ros/benchmarks/src/BenchmarkExecutor.cpp

```diff
@@ -481,9 +490,12 @@ void BenchmarkExecutor::createRequestCombinations(const BenchmarkRequest& breque
       // Duplicate the request for each of the path constraints
       for (const PathConstraints& path_constraint : path_constraints)
       {
-        new_brequest.request.path_constraints = path_constraint.constraints[0];
-        new_brequest.name = start_state.name + "_" + new_brequest.name + "_" + path_constraint.name;
-        requests.push_back(new_brequest);
+        if (start_state.name != path_constraint.name)
```

Comment:
Actually, this was misplaced and was intended to filter out start/goal state pairs that have the same name (in preparation of https://github.com/ros-planning/moveit/pull/1548).
I will remove this since it doesn't belong in this PR